cypress/support/e2e.js

```diff
@@ -3,4 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import '../utils/commands';
+import '../utils/commands.js';
+import '../utils/dashboards/data_explorer/commands.js';
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
+
+// eslint-disable-next-line no-unused-vars
+Cypress.on('uncaught:exception', (_err) => {
+  // returning false here prevents Cypress from failing the test
+  return false;
+});
```

Comment:
Makes sense, there are some uncaught errors that we do want to gracefully ignore. That being said, I feel there are times when bubbling up an uncaught exception would be really useful. For example, if a page fails to render it'd be nice to see the exception in the runner rather than having to run locally and dig for it.

Can we be a bit more selective here and only swallow the error you were seeing needed to be handled gracefully?

Ex.
```
const resizeObserverLoopErr = /^[^(ResizeObserver loop limit exceeded)]/;
Cypress.on('uncaught:exception', (err) => {
  /* returning false here prevents Cypress from failing the test */
  if (resizeObserverLoopErr.test(err.message)) {
    return false;
  }
});
```

Reply:
Sounds good. I added this as I was getting a specific navigation error. Keeping the ignore selective makes a lot of sense to me.